util/help/topics.go

```diff
@@ -1,13 +1,35 @@
 package help
 
-const (
-	root       = "https://argo-workflows.readthedocs.io/en/latest"
```

Comment:
> Raised [in response to this](https://github.com/argoproj/argo-workflows/pull/13265#discussion_r1699400713)

For reference, this was versioned before as well -- each release branch changes the tail of its `root`. 
See also #12446, #12462, #12473.

That's why I changed the title to say "dynamically get", as before it was statically referenced. The scope is also for the CLI.

Note that per the above PRs, there are still several other places that use static version links that could be updated to use this dynamic variant instead.
Some won't be updateable as they're in static documentation (e.g. example YAML, swagger, direct README and other direct markdown as opposed to docs website rendered, etc). UI would also need similar code to pull out the version numbers to make its references dynamic

Reply:
Yeah, this is this story: I forgot this existed at all when writing #13265, despite me having used it before in #11646 - so #13265 wasn't using this, and I wrote this function and pulled it out into this PR.

Anyway, I think we're in agreement this is a good change, so I'll fix up some of it in #13265.